### ISSUE TITLE
Accessibility: Contrast is too low between secondary brand colors and white

### ISSUE THREAD

**Issue by jrchamp** [openstax/webview#2285]
"Get This Book!", "Introduction" and footer icons have insufficient contrast
See: https://cnx.org/contents/Rko_umjB@6.4:MXTQeFW9@5/Introduction

The green color comes from here:
https://github.com/openstax/webview/blob/b7838e4422a1235d15caf078b68425a525a0f98b/src/styles/variables.less#L11-L12

Please note that both the "secondary" and "secondary-dim" are too similar to the white background, failing to meet the minimum contrast of 4.5:1 for WCAG 2.0 AA compliance.
